src/EventTickets/resources/components/EventTicketsList.tsx

```diff
@@ -0,0 +1,71 @@
+import {useEffect, useState} from 'react';
+import EventTicketsListItem from './EventTicketsListItem';
+import useCurrencyFormatter from '@givewp/forms/app/hooks/useCurrencyFormatter';
+
+export default function EventTicketsList({tickets, ticketsLabel, soldOutMessage, config}) {
+    const [selectedTickets, setSelectedTickets] = useState({});
+    const {currency, useDonationSummary, setValue} = config;
+    const formatter = useCurrencyFormatter(currency);
+    const donationSummary = useDonationSummary && useDonationSummary();
```

Comment:
What is the use case for sometimes using Donation Summary and sometimes not? This component is doing a number of different things based on a configuration, which suggests that it is doing too much.

Reply:
Refactored here: 059fd69cee4b77a7e89e619d330609849d50ad9f